fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java

```diff
@@ -62,7 +59,7 @@ public Boolean hasReturn() {
      * @return true if the header has a return tag
      */
     public Boolean hasHandling() {
-        return any(preferTags(), getPreferTag("handling"));
+        return preferTags().stream().anyMatch(x -> x.getTag().equals("handling"));
```

Comment:
No dice on that -- the function returns a PreferTag and needs to filter on a String, so we can't map to a string and expect to get a PreferTag back.

> On Jun 2, 2015, at 12:29 PM, A. Soroka <notifications@github.com> wrote:
> 
> In fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java:
> 
> > @@ -62,7 +59,7 @@ public Boolean hasReturn() {
> >       * @return true if the header has a return tag
> >       */
> >      public Boolean hasHandling() {
> > -        return any(preferTags(), getPreferTag("handling"));
> > +        return preferTags().stream().anyMatch(x -> x.getTag().equals("handling"));
> 
> Again, maybe .map(PreferTag::getTag).anyMatch("handling"::equals)?
> 
> —
> Reply to this email directly or view it on GitHub.
>